src/core/binding.c

```diff
@@ -318,13 +318,17 @@ QuicBindingRegisterListener(
 
         if (NewFamily > ExistingFamily) {
             break; // End of possible family matches. Done searching.
-        } else if (NewFamily != ExistingFamily) {
```

Comment:
Does clang actually complain about this?

Reply:
I'm definitely not a big fan of replacing all else ifs with else statements, which it looks like this PR does a lot of.

Reply:
I actually don't mind those changes. After looking at the code a while, I think I might like it better.

Reply:
The `else if` -> `if` changes address the [readability-else-after-return](https://clang.llvm.org/extra/clang-tidy/checks/readability-else-after-return.html) and similar warning. I prefer it, but we can also simply disable those checks.